CRM/Core/I18n.php

```diff
@@ -163,14 +163,22 @@ public static function languages($justEnabled = FALSE) {
     if (!$all) {
       $all = CRM_Contact_BAO_Contact::buildOptions('preferred_language');
 
+      // get labels
+      $rows = array();
+      $labels = array();
+      CRM_Core_OptionValue::getValues(array('name' => 'languages'), $rows);
```

Comment:
@mlutfy @samuelsov In interregnum between the original PR and this revised submission, I think this function was deprecated. On my local dev system, it's showing more warnings like this:

```
Jun 20 00:02:43  [warning] Deprecated function, use CRM_Core_PseudoConstant::getKey
Array
(
    [civi.tag] => deprecated
)
```